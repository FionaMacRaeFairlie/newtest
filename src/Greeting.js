// Greeting.js
import React from 'react';


const Greeting = ({ name }) => (
  <p>Hello, {name}!</p>
);

export default Greeting; 
