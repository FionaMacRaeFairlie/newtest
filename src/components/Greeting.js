// Greeting.js
import React from 'react';


const Greeting = ({ name }) => (
  <div className="jumbotron">
    <h1 className="display-4">Hello, {name}!</h1>
  </div>
);

export default Greeting; 
