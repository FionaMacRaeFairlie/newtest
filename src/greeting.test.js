import React from 'react';
import renderer from 'react-test-renderer';

import Greeting from './components/Greeting';

it('renders correctly when there are no items', () => {
const tree = renderer.create(<Greeting />).toJSON();
expect(tree).toMatchSnapshot();
});