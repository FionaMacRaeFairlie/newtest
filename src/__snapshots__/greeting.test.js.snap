// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
<div
  className="jumbotron"
>
  <h1
    className="display-4"
  >
    Hello, 
    !
  </h1>
</div>
`;
